### ISSUE TITLE
Supervisor has no Internet

### ISSUE THREAD

**Issue by Jaum3** [home-assistant/supervisor#3790]
### Describe the issue you are experiencing

I've tried to Install HA on a new Raspberry Pi4 B this week.
I've used the Raspberry Pi Official Imager since I've had a few problems with balena etcher...

I can acces homeassistant.local:8123 but each time I have an error.
First, It's telling me that the Supervisor has no Internet. So, with the help of another issue reported, i changed my Ip configuration...

Now, the loggs are telling me that my supervisor has no internet still and that my system is unhealthy...

Can you help me please ?

### What is the used version of the Supervisor?

Can't see it

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

Can't see it

### What version of Home Assistant Core is installed?

Can't see it

### Steps to reproduce the issue

1. Install HA on a SD card.
2. Open homeassistant.local:8123
3. Wait
4. Check the logs 

### Anything in the Supervisor logs that might be useful for us?

```s6-rc: info: service s6rc-oneshot-runner: starting s6-rc: info: service s6rc-oneshot-runner successfully started s6-rc: info: service fix-attrs: starting s6-rc: info: service fix-attrs successfully started s6-rc: info: service legacy-cont-init: starting cont-init: info: running /etc/cont-init.d/udev.sh [07:35:10] INFO: Using udev information from host cont-init: info: /etc/cont-init.d/udev.sh exited 0 s6-rc: info: service legacy-cont-init successfully started s6-rc: info: service legacy-services: starting services-up: info: copying legacy longrun supervisor (no readiness notification) services-up: info: copying legacy longrun watchdog (no readiness notification) s6-rc: info: service legacy-services successfully started [07:35:10] INFO: Starting local supervisor watchdog... 22-03-11 07:35:15 INFO (MainThread) [__main__] Initializing Supervisor setup 22-03-11 07:35:15 INFO (MainThread) [supervisor.docker.network] Can't find Supervisor network, creating a new network 22-03-11 07:35:16 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry 22-03-11 07:35:16 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: raspberrypi4-64 22-03-11 07:35:16 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor homeassistant/aarch64-hassio-supervisor with version 2022.07.0 22-03-11 07:35:16 INFO (SyncWorker_0) [supervisor.docker.supervisor] Connecting Supervisor to hassio-network 22-03-11 07:35:16 WARNING (SyncWorker_0) [supervisor.docker.interface] Can't find homeassistant/aarch64-hassio-supervisor for cleanup 22-03-11 07:35:16 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE 22-03-11 07:35:16 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete 22-03-11 07:35:16 INFO (MainThread) [__main__] Setting up Supervisor 22-03-11 07:35:21 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2 22-03-11 07:35:21 INFO (MainThread) [supervisor.hardware.monitor] Started Supervisor hardware monitor 22-03-11 07:35:21 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface io.hass.os 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.systemd1 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.login1 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.hostname1 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.timedate1 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.NetworkManager 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface de.pengutronix.rauc 22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.resolve1 22-03-11 07:35:22 INFO (MainThread) [supervisor.host.info] Updating local host information 22-03-11 07:35:22 INFO (MainThread) [supervisor.host.services] Updating service information 22-03-11 07:35:22 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information 22-03-11 07:35:22 INFO (MainThread) [supervisor.host.manager] Host information reload completed 22-03-11 07:35:22 INFO (MainThread) [supervisor.host.network] Updating local network information 22-03-11 07:40:39 INFO (MainThread) [supervisor.host.network] Updating local network information 22-03-11 07:45:20 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: {'hassio-supervisor'} 22-03-11 07:45:25 WARNING (MainThread) [supervisor.utils.whoami] Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again] 22-03-11 07:45:25 WARNING (MainThread) [supervisor.core] Can't adjust Time/Date settings: Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again] 22-03-11 07:45:25 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-cli versions: [<AwesomeVersion CalVer '2022.06.0'>] 22-03-11 07:45:25 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-cli with version 2022.06.0 22-03-11 07:45:25 INFO (MainThread) [supervisor.plugins.cli] Starting CLI plugin 22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.cli] Starting CLI ghcr.io/home-assistant/aarch64-hassio-cli with version 2022.06.0 - 172.30.32.5 22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-dns versions: [<AwesomeVersion CalVer '2022.04.1'>] 22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-dns with version 2022.04.1 22-03-11 07:45:26 INFO (MainThread) [supervisor.plugins.dns] Starting CoreDNS plugin 22-03-11 07:45:28 INFO (SyncWorker_0) [supervisor.docker.dns] Starting DNS ghcr.io/home-assistant/aarch64-hassio-dns with version 2022.04.1 - 172.30.32.3 22-03-11 07:45:28 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf 22-03-11 07:45:33 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-audio versions: [<AwesomeVersion CalVer '2022.05.0'>] 22-03-11 07:45:33 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-audio with version 2022.05.0 22-03-11 07:45:33 INFO (MainThread) [supervisor.plugins.audio] Starting Audio plugin 22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.audio] Starting Audio ghcr.io/home-assistant/aarch64-hassio-audio with version 2022.05.0 - 172.30.32.4 22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-observer versions: [<AwesomeVersion CalVer '2021.10.0'>] 22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0 22-03-11 07:45:34 INFO (MainThread) [supervisor.plugins.observer] Starting observer plugin 22-03-11 07:45:36 INFO (SyncWorker_0) [supervisor.docker.observer] Starting Observer ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0 - 172.30.32.6 22-03-11 07:45:36 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-multicast versions: [<AwesomeVersion CalVer '2022.02.0'>] 22-03-11 07:45:36 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0 22-03-11 07:45:36 INFO (MainThread) [supervisor.plugins.multicast] Starting Multicast plugin 22-03-11 07:45:37 INFO (SyncWorker_0) [supervisor.docker.multicast] Starting Multicast ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0 - Host 22-03-11 07:45:37 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection 22-03-11 07:45:37 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection 22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 0 Home Assistant secrets 22-03-11 07:45:37 INFO (SyncWorker_1) [supervisor.docker.interface] No version found for ghcr.io/home-assistant/raspberrypi4-64-homeassistant 22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] No Home Assistant Docker image ghcr.io/home-assistant/raspberrypi4-64-homeassistant found. 22-03-11 07:45:37 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/raspberrypi4-64-homeassistant with version landingpage 22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] Using preinstalled landingpage 22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] Starting HomeAssistant landingpage 22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.module] Update pulse/client.config: /data/tmp/homeassistant_pulse 22-03-11 07:45:38 INFO (SyncWorker_0) [supervisor.docker.homeassistant] Starting Home Assistant ghcr.io/home-assistant/raspberrypi4-64-homeassistant with version landingpage 22-03-11 07:45:38 INFO (MainThread) [supervisor.os.manager] Detect Home Assistant Operating System 8.4 / BootSlot A 22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'StoreManager.update_repositories' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Found 0 installed add-ons 22-03-11 07:45:38 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files 22-03-11 07:45:38 INFO (MainThread) [supervisor.discovery] Loaded 0 messages 22-03-11 07:45:38 INFO (MainThread) [supervisor.ingress] Loaded 0 ingress sessions 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] System checks complete 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete 22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete 22-03-11 07:45:38 INFO (MainThread) [__main__] Running Supervisor 22-03-11 07:45:38 INFO (MainThread) [supervisor.os.manager] Rauc: A - marked slot kernel.0 as good 22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons 22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SYSTEM' starting 0 add-ons 22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SERVICES' starting 0 add-ons 22-03-11 07:45:38 INFO (MainThread) [supervisor.core] Skiping start of Home Assistant 22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.APPLICATION' starting 0 add-ons 22-03-11 07:45:38 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled 22-03-11 07:45:38 INFO (MainThread) [supervisor.core] Supervisor is up and running 22-03-11 07:45:38 INFO (MainThread) [supervisor.homeassistant.core] Home Assistant setup 22-03-11 07:45:38 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec 22-03-11 07:45:38 INFO (MainThread) [supervisor.host.info] Updating local host information 22-03-11 07:45:38 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER 22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckDNSServerIPv6Errors.run_check' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON 22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckAddonPwned.run_check' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER 22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckDNSServerFailures.run_check' blocked from execution, no supervisor internet connection 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.CREATE_FULL_BACKUP - ContextType.SYSTEM / None 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] System checks complete 22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING 22-03-11 07:45:39 INFO (MainThread) [supervisor.host.services] Updating service information 22-03-11 07:45:39 INFO (MainThread) [supervisor.host.network] Updating local network information 22-03-11 07:45:53 WARNING (MainThread) [supervisor.utils.codenotary] Timeout while processing CodeNotary 22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.TRUST - ContextType.SYSTEM / None 22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_INTEGRITY - ContextType.SYSTEM / None 22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete 22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING 22-03-11 07:45:53 WARNING (MainThread) [supervisor.jobs] 'FixupSystemExecuteIntegrity.process_fixup' blocked from execution, no supervisor internet connection 22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete 22-03-11 07:45:56 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information 22-03-11 07:45:56 INFO (MainThread) [supervisor.host.manager] Host information reload completed 22-03-11 07:46:08 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection 22-03-11 07:46:08 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec
```

**Comment by mdegat01** [comment]
> Now, the loggs are telling me that my supervisor has no internet still and that my system is unhealthy...

What are these logs? Can you share them? The supervisor logs not the core ones.

**Comment by Jaum3** [comment]
Here's a small part of the logs. The rest is the same as the lise 2 lines... 

`s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/udev.sh
[07:35:10] INFO: Using udev information from host
cont-init: info: /etc/cont-init.d/udev.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun supervisor (no readiness notification)
services-up: info: copying legacy longrun watchdog (no readiness notification)
s6-rc: info: service legacy-services successfully started
[07:35:10] INFO: Starting local supervisor watchdog...
22-03-11 07:35:15 INFO (MainThread) [__main__] Initializing Supervisor setup
22-03-11 07:35:15 INFO (MainThread) [supervisor.docker.network] Can't find Supervisor network, creating a new network
22-03-11 07:35:16 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry
22-03-11 07:35:16 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: raspberrypi4-64
22-03-11 07:35:16 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor homeassistant/aarch64-hassio-supervisor with version 2022.07.0
22-03-11 07:35:16 INFO (SyncWorker_0) [supervisor.docker.supervisor] Connecting Supervisor to hassio-network
22-03-11 07:35:16 WARNING (SyncWorker_0) [supervisor.docker.interface] Can't find homeassistant/aarch64-hassio-supervisor for cleanup
22-03-11 07:35:16 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE
22-03-11 07:35:16 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-03-11 07:35:16 INFO (MainThread) [__main__] Setting up Supervisor
22-03-11 07:35:21 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
22-03-11 07:35:21 INFO (MainThread) [supervisor.hardware.monitor] Started Supervisor hardware monitor
22-03-11 07:35:21 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface io.hass.os
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.systemd1
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.login1
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.hostname1
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.timedate1
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.NetworkManager
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface de.pengutronix.rauc
22-03-11 07:35:22 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.resolve1
22-03-11 07:35:22 INFO (MainThread) [supervisor.host.info] Updating local host information
22-03-11 07:35:22 INFO (MainThread) [supervisor.host.services] Updating service information
22-03-11 07:35:22 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
22-03-11 07:35:22 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-03-11 07:35:22 INFO (MainThread) [supervisor.host.network] Updating local network information
22-03-11 07:40:39 INFO (MainThread) [supervisor.host.network] Updating local network information
22-03-11 07:45:20 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: {'hassio-supervisor'}
22-03-11 07:45:25 WARNING (MainThread) [supervisor.utils.whoami] Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again]
22-03-11 07:45:25 WARNING (MainThread) [supervisor.core] Can't adjust Time/Date settings: Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again]
22-03-11 07:45:25 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-cli versions: [<AwesomeVersion CalVer '2022.06.0'>]
22-03-11 07:45:25 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-cli with version 2022.06.0
22-03-11 07:45:25 INFO (MainThread) [supervisor.plugins.cli] Starting CLI plugin
22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.cli] Starting CLI ghcr.io/home-assistant/aarch64-hassio-cli with version 2022.06.0 - 172.30.32.5
22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-dns versions: [<AwesomeVersion CalVer '2022.04.1'>]
22-03-11 07:45:26 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-dns with version 2022.04.1
22-03-11 07:45:26 INFO (MainThread) [supervisor.plugins.dns] Starting CoreDNS plugin
22-03-11 07:45:28 INFO (SyncWorker_0) [supervisor.docker.dns] Starting DNS ghcr.io/home-assistant/aarch64-hassio-dns with version 2022.04.1 - 172.30.32.3
22-03-11 07:45:28 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf
22-03-11 07:45:33 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-audio versions: [<AwesomeVersion CalVer '2022.05.0'>]
22-03-11 07:45:33 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-audio with version 2022.05.0
22-03-11 07:45:33 INFO (MainThread) [supervisor.plugins.audio] Starting Audio plugin
22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.audio] Starting Audio ghcr.io/home-assistant/aarch64-hassio-audio with version 2022.05.0 - 172.30.32.4
22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-observer versions: [<AwesomeVersion CalVer '2021.10.0'>]
22-03-11 07:45:34 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0
22-03-11 07:45:34 INFO (MainThread) [supervisor.plugins.observer] Starting observer plugin
22-03-11 07:45:36 INFO (SyncWorker_0) [supervisor.docker.observer] Starting Observer ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0 - 172.30.32.6
22-03-11 07:45:36 INFO (SyncWorker_0) [supervisor.docker.interface] Found ghcr.io/home-assistant/aarch64-hassio-multicast versions: [<AwesomeVersion CalVer '2022.02.0'>]
22-03-11 07:45:36 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0
22-03-11 07:45:36 INFO (MainThread) [supervisor.plugins.multicast] Starting Multicast plugin
22-03-11 07:45:37 INFO (SyncWorker_0) [supervisor.docker.multicast] Starting Multicast ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0 - Host
22-03-11 07:45:37 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection
22-03-11 07:45:37 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection
22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 0 Home Assistant secrets
22-03-11 07:45:37 INFO (SyncWorker_1) [supervisor.docker.interface] No version found for ghcr.io/home-assistant/raspberrypi4-64-homeassistant
22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] No Home Assistant Docker image ghcr.io/home-assistant/raspberrypi4-64-homeassistant found.
22-03-11 07:45:37 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/raspberrypi4-64-homeassistant with version landingpage
22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] Using preinstalled landingpage
22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.core] Starting HomeAssistant landingpage
22-03-11 07:45:37 INFO (MainThread) [supervisor.homeassistant.module] Update pulse/client.config: /data/tmp/homeassistant_pulse
22-03-11 07:45:38 INFO (SyncWorker_0) [supervisor.docker.homeassistant] Starting Home Assistant ghcr.io/home-assistant/raspberrypi4-64-homeassistant with version landingpage
22-03-11 07:45:38 INFO (MainThread) [supervisor.os.manager] Detect Home Assistant Operating System 8.4 / BootSlot A
22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'StoreManager.update_repositories' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Found 0 installed add-ons
22-03-11 07:45:38 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files
22-03-11 07:45:38 INFO (MainThread) [supervisor.discovery] Loaded 0 messages
22-03-11 07:45:38 INFO (MainThread) [supervisor.ingress] Loaded 0 ingress sessions
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-03-11 07:45:38 INFO (MainThread) [__main__] Running Supervisor
22-03-11 07:45:38 INFO (MainThread) [supervisor.os.manager] Rauc: A - marked slot kernel.0 as good
22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SYSTEM' starting 0 add-ons
22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SERVICES' starting 0 add-ons
22-03-11 07:45:38 INFO (MainThread) [supervisor.core] Skiping start of Home Assistant
22-03-11 07:45:38 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.APPLICATION' starting 0 add-ons
22-03-11 07:45:38 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
22-03-11 07:45:38 INFO (MainThread) [supervisor.core] Supervisor is up and running
22-03-11 07:45:38 INFO (MainThread) [supervisor.homeassistant.core] Home Assistant setup
22-03-11 07:45:38 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec
22-03-11 07:45:38 INFO (MainThread) [supervisor.host.info] Updating local host information
22-03-11 07:45:38 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckDNSServerIPv6Errors.run_check' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckAddonPwned.run_check' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
22-03-11 07:45:38 INFO (MainThread) [supervisor.jobs] 'CheckDNSServerFailures.run_check' blocked from execution, no supervisor internet connection
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.CREATE_FULL_BACKUP - ContextType.SYSTEM / None
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-03-11 07:45:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-03-11 07:45:39 INFO (MainThread) [supervisor.host.services] Updating service information
22-03-11 07:45:39 INFO (MainThread) [supervisor.host.network] Updating local network information
22-03-11 07:45:53 WARNING (MainThread) [supervisor.utils.codenotary] Timeout while processing CodeNotary
22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.module] Create new issue IssueType.TRUST - ContextType.SYSTEM / None
22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.EXECUTE_INTEGRITY - ContextType.SYSTEM / None
22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-03-11 07:45:53 WARNING (MainThread) [supervisor.jobs] 'FixupSystemExecuteIntegrity.process_fixup' blocked from execution, no supervisor internet connection
22-03-11 07:45:53 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
22-03-11 07:45:56 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
22-03-11 07:45:56 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-03-11 07:46:08 WARNING (MainThread) [supervisor.jobs] 'Updater.fetch_data' blocked from execution, no supervisor internet connection
22-03-11 07:46:08 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec`

**Comment by Jaum3** [comment]
I don't know if this are the supervisor logs or the core one but I don't know how to download them. I just copy and pasted them in there.

**Comment by mdegat01** [comment]
You got the right logs. Although very tough to read, not sure what happened to all the newlines. Anyway this is what I was looking for:
```
22-03-11 07:45:25 WARNING (MainThread) [supervisor.utils.whoami] Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again]
22-03-11 07:45:25 WARNING (MainThread) [supervisor.core] Can't adjust Time/Date settings: Can't fetch Whoami data: Cannot connect to host services.home-assistant.io:443 ssl:default [Try again]
```
Believe that makes it a duplicate of #3764 that should also be fixed by #3783 . Until that fix makes it to a supervisor release I believe a user found a workaround [here](https://community.home-assistant.io/t/fresh-installation-stuck-on-preparing-ha/446308) by setting the date manually.

**Comment by Jaum3** [comment]
Thank you for helping me out! I'll tell you if I have any more problems!
